phoenicis-dist/src/scripts/phoenicis-create-package.sh

```diff
@@ -62,7 +62,45 @@ fi
 
 if [ "$PHOENICIS_OPERATING_SYSTEM" == "Linux" ]; then
     jpackager create-image "${PHOENICIS_JPACKAGER_ARGUMENTS[@]}"  --linux-bundle-name "phoenicis-playonlinux"
-    jpackager create-installer deb "${PHOENICIS_JPACKAGER_ARGUMENTS[@]}" --linux-package-deps "unzip, wget, xterm | x-terminal-emulator, imagemagick, cabextract, icoutils, p7zip-full, curl, winbind" --linux-deb-maintainer "PlayOnLinux Packaging <packages@playonlinux.com>" --linux-bundle-name "phoenicis-playonlinux"
```

Comment:
What about `rpm` installer?
As far as I understand `jpackager` it is also able to create `rpm` installer

Reply:
jpackager does no longer create the .deb. I do it manually, it's much more flexible.
The rpm will be done the same way as we did in for POLv4, maybe in a separate ticket